packages/orama/src/methods/search.ts

```diff
@@ -191,18 +191,21 @@ export async function search(orama: Orama, params: SearchParams, language?: stri
     )
   }
 
-  // Get unique doc IDs from uniqueDocsIDs map, sorted by value.
-  let uniqueDocsArray = Object.entries(context.uniqueDocsIDs).sort(sortTokenScorePredicate)
+  // Get unique doc IDs from uniqueDocsIDs map
+  let uniqueDocsArray = Object.entries(context.uniqueDocsIDs)
 
   // If filters are enabled, we need to remove the IDs of the documents that don't match the filters.
   if (hasFilters) {
     uniqueDocsArray = intersectFilteredIDs(whereFiltersIDs, uniqueDocsArray)
   }
 
-  const resultIDs: Set<string> = new Set()
-  // Populate facets if needed
-  const facets = shouldCalculateFacets ? await getFacets(orama, uniqueDocsArray, params.facets!) : {}
+  if (params.sortBy) {
+    uniqueDocsArray = await orama.sorter.sortBy(orama.data.sorter, uniqueDocsArray, params.sortBy)
+  } else {
+    uniqueDocsArray = uniqueDocsArray.sort(sortTokenScorePredicate)
```

Comment:
@allevo is this where users can define their own sorting predicate?

Reply:
Now it is allowed. The user can specify a custom sort function